testdata/event_listener

```diff
@@ -183,7 +183,7 @@ compact         1   2.3 K     0 B       0          (size == estimated-debt, scor
 zmemtbl         0     0 B
    ztbl         0     0 B
  bcache         8   1.4 K    5.9%  (score == hit-rate)
- tcache         1   616 B    0.0%  (score == hit-rate)
+ tcache         1   640 B    0.0%  (score == hit-rate)
```

Comment:
These extra 24 bytes comes from the 16 bytes in the new range key block handle and the 8 bytes for the uint64 for the range key counter in the metadata.